src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp

```diff
@@ -2301,7 +2305,9 @@ void FixedwingPositionControl::tecs_update_pitch_throttle(float alt_sp, float v_
 	// we're in transition
 	if (_vehicle_status.is_vtol && _vehicle_status.in_transition_mode) {
 		_was_in_transition = true;
-		_asp_after_transition = _ctrl_state.airspeed;
+		// set this to transition airspeed to init tecs correctly
+		// some vtols fly without airspeed sensor
+		_asp_after_transition = _parameters.airspeed_trans;
```

Comment:
The reason this logic is here to ramp up desired airspeed so there is no stop and TECS doesn't try something stupid like pitch down and throttle up. Please use the minimum airspeed (not transition) as lower bound initialization value, then it should be fine